### DISCUSSION TITLE
Cpu load

### DISCUSSION THREAD

**Discussion by Lucky390163** [moneytoo/Player#442] [General]
Can u reduce cpu load of app.

**Comment by moneytoo** [comment]
No. But there shouldn't be any CPU load anyway. Almost all video decoding should be hardware accelerated. Possibly only some unsupported audio formats may be decoded on CPU (like DTS), but that is done by the FFMpeg plugin and there's not much to do with that.

**Comment by Lucky390163** [comment]
Sir i have a big issue regarding video matroska formats are not playing even loading the circle keep on rotates please take a look.

720P mkv file audio format matroska.

**Comment by moneytoo** [comment]
It really depends on the actual file and device capabilities. Share more details about your device and media (sample or MediaInfo output).

**Comment by Lucky390163** [comment]
![Screenshot_20230306_212819_VLC.jpg](https://user-images.githubusercontent.com/115852216/223163294-5ff9e432-91ff-4883-9049-53ca2df9b067.jpg)

Here it is.

**Comment by Lucky390163** [comment]
Iam sending a video in your email pleae check.

**Comment by Lucky390163** [comment]
Here if is.
